docs/quickstart.md

```diff
@@ -0,0 +1,189 @@
+
+### Test service with a sandbox
+The CDS hooks initiative provides a publicly available [sandbox](http://sandbox.cds-hooks.org/) to test your service. 
+
+Select the configure hooks:<br>
+![Demo Configuration](images/demo-configure-hooks.png)
+
+Delete the existing hooks, and then do a quick add with a reference to your CDS service:<br>
+![Patient View Hooks Launch from Sandbox](images/demo-quick-add.png)
+
+After testing with the sandbox, you are ready to connect with an EHR service.
+
+## Building an EHR Service
+Build out following sections:
+
+1. Calls discovery endpoint 
+1. Invoke service on patient-view 
+1. Support for FHIR resources on request (context or pre-fetch)
+1. Exposed non-secured FHIR server
+1. Render card
+1. Launch SMART app 
+1. Tested with external CDS service
+
+This tutorial recommends implementing the [security](specification/#security) after succesful open access testing.
+
+### Calls discovery endpoint 
+The CDS discovery endpoint provides the list of services a CDS provider supports, and the hooks a service should be invoked on. An EHR may configure their system to support a set of hooks at a certain location in their users work flow, or build a dynamic capability to interact with a CDS Service provider within a work flow. For the best end-user experience, this guide recommends a business analyst configure which hooks an EHR will support. 
+
+Below is an example work flow where a business analyst accesses this list of available services by calling 
+
+`GET https://example.com/cds-services` 
+
+and then configures them in the system. 
+
+![business analyst configuration](images/analyst-configuration-spec.png)
+
+This image captures a business analyst reviewing services from one CDS provider. A business analyst may review services from multiple providers and configure appropriate services per user profiles.
+
+### Invoke service on patient-view hook
+The patient-view hook is invoked when a patient chart is opened. It's one of the most basic since the logic doesn't have any prior workflow dependencies. The service called on the patient-view hook could be dependent on patient characteristics, for example: sex, problems in problems list, active medications, etc. The current version of the CDS Hooks specification allows the EHR to decide which characteristics to consider. 
+
+### Support for FHIR resources on request or prefetch
+Often a CDS service will require additional information from the EHR to perform the decision support logic, or determine the appropriate SMART app to return. Prefetch provides the EHR the capability to pass a resource when invoking a service. For example, with a patient resource included a service could do a geography search for potential environmental risk factors. Below is an example request invoked on patient-view with a patient included: (***fix server, not working for me)
```

Comment:
eek, can't immediately remember.  My guess is I couldn't get the http://hooks.smarthealthit.org to work -- do we have a preferred FHIR server, and CDS server for examples?

We can fix in all location later since this server is referenced other places. I will remove this specific comment.

Reply:
Today our example patient-view CDS Service is available at http://hooks.fhir.me:8082/cds-services/patient-hello-world. However, @zplata is working to replace this over in [cds-hooks/sandbox-cds-services](https://github.com/cds-hooks/sandbox-cds-services) so that URL will be changing very soon.